bin/translations_upload.ts

```diff
@@ -17,10 +17,10 @@
  *
  */
 
-const {join, resolve} = require('path');
+const path = require('path');
 const {execSync} = require('child_process');
 
-const root = resolve(__dirname, '..');
-const config = join(root, 'crowdin.yaml');
-const identity = join(root, 'keys', 'crowdin.yaml');
+const root = path.resolve(__dirname, '..');
```

Comment:
`resolve()` is easily confused with `Promise.resolve()`, that's why I decided to import `path` completely.